### PR TITLE
chore: bump to 2.0.0

### PR DESCRIPTION
**Pull request by henryiii**
- chore: drop unused py file
- chore: bump version to 2.0.0

### CODE REVIEW

**Comment by henryiii** [comment]
@phlptp Waiting for a sign-off from you that we are good to release.

**Review by phlptp — APPROVED** [review]
I am good with this.  The changelog and the changes here look good.    I haven't looked through the book completely yet, but that can wait I think If any changes are needed.    So I think it is time for a release.

**Pull request merged**
